### PR TITLE
[Improve][Doc]Update Kafka source connector docs

### PR DESCRIPTION
**Pull request by Huanli-Meng**
### Motivation

The Kafka source connector configs are updated. Therefore, update docs accordingly.

### Modifications

- Update Kafka source connector docs.
- Related releases:
  - latest
  - 2.10.0
  - 2.9.2

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `no-need-doc` 
(Please explain why)
  
- [ x ] `doc` 
(Your PR contains doc changes)

- [ ] `doc-added`
(Docs have been already added)

### CODE REVIEW

**Review by Anonymitaet — APPROVED** [review]

**Comment by Huanli-Meng** [comment]
@freeznet could you please help review this PR? Thanks.

**Comment by Anonymitaet** [comment]
@Huanli-Meng can you rename your PR? eg. [improve][doc] xxx
guideline: https://docs.google.com/document/d/1d8Pw6ZbWk-_pCKdOmdvx9rnhPiyuxwq60_TrD68d7BA/edit#

File: site2/website/versioned_docs/version-2.9.2/io-kafka-source.md [hunk 1]
```diff
@@ -182,18 +163,49 @@ Here is an example of using the Kafka source connector with the configuration fi
...
+    PULSAR_HOME/bin/pulsar-admin sources reload
+    ```
+
+3. Check whether the Kafka source connector is available on the list or not.
+
+    ```
+    PULSAR_HOME/bin/pulsar-admin sources available-sources
+    ```
+
+4. Create a Kafka source connector on a Pulsar cluster using the [`pulsar-admin sources create`](http://pulsar.apache.org/tools/pulsar-admin/2.8.0-SNAPSHOT/#-em-create-em--14) command.
```
Comment (freeznet) [inline site2/website/versioned_docs/version-2.9.2/io-kafka-source.md:206]: maybe link to current version's doc?

File: site2/website/versioned_docs/version-2.10.0/io-kafka-source.md [hunk 2]
```diff
@@ -184,18 +163,49 @@ Here is an example of using the Kafka source connector with the configuration fi
...
+    PULSAR_HOME/bin/pulsar-admin sources reload
+    ```
+
+3. Check whether the Kafka source connector is available on the list or not.
+
+    ```
+    PULSAR_HOME/bin/pulsar-admin sources available-sources
+    ```
+
+4. Create a Kafka source connector on a Pulsar cluster using the [`pulsar-admin sources create`](http://pulsar.apache.org/tools/pulsar-admin/2.8.0-SNAPSHOT/#-em-create-em--14) command.
```
Comment (freeznet) [inline site2/website/versioned_docs/version-2.10.0/io-kafka-source.md:206]: maybe link to current version's doc?

**Review by freeznet — APPROVED** [review]

File: site2/website/versioned_docs/version-2.9.2/io-kafka-source.md (hunk 1 above)
Comment (Huanli-Meng) [reply site2/website/versioned_docs/version-2.9.2/io-kafka-source.md]: Good point. Have fixed

File: site2/website/versioned_docs/version-2.10.0/io-kafka-source.md (hunk 2 above)
Comment (Huanli-Meng) [reply site2/website/versioned_docs/version-2.10.0/io-kafka-source.md]: Good point. Have fixed

**Pull request merged**